src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/Dependencies/ResolvedProjectReference.xaml

```diff
@@ -30,9 +30,7 @@
                   ReadOnly="True"
                   Visible="False">
     <StringProperty.DataSource>
-      <DataSource ItemType="ProjectReference"
-                  PersistedName="Identity"
```

Comment:
why were these two properties removed, ItemType and Persistence?

Reply:
From memory this change makes the returned path be absolute rather than relative. Previously it returned the value of the `<ProjectReference Include="../OtherProject/OtherProject.csproj"/>` rather than the computed resolved path. We want the resolved path for the Object Browser.